src/syntax/sugar.rkt

```diff
@@ -153,20 +150,17 @@
          (define-values (iff* iff-repr) (loop iff ctx))
          (values `(if ,cond* ,ift* ,iff*) ift-repr)]
         [`(! ,props ... ,body)
-         (define ctx*
-           (match (dict-ref (props->dict props) ':props #f)
-             [#f ctx]
-             [prec (struct-copy context ctx [repr (get-representation prec)])]))
-         (loop body ctx*)]
+         (define props* (props->dict props))
+         (loop body
+               (match (dict-ref props* ':precision #f)
```

Comment:
Is changing `:props` to `:precision` here a bug fix?

Reply:
Yeah can't believe how long that has been in there